core/arch/arm/kernel/boot.c

```diff
@@ -1512,3 +1581,38 @@ void __weak boot_save_args(unsigned long a0, unsigned long a1 __unused,
 		}
 	}
 }
+
+#if defined(CFG_TRANSFER_LIST)
+static TEE_Result release_transfer_list(void)
+{
+	struct dt_descriptor *dt = get_external_dt_desc();
+
+	if (!mapped_tl)
+		return TEE_SUCCESS;
+
+	if (dt) {
```

Comment:
Please add a comment describing that `release_external_dt()` will do nothing if the external DT is located in the memory area of a transfer list since it uses a different memtype.

Reply:
I added new comments in `release_external_dt()`, `init_external_dt()` and `release_transfer_list()` to increase the readibility.